WBooks/SuggestionCollectionViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SuggestionCollectionViewController.swift
+//  WBooks
+//
+//  Created by Nicolas Manograsso on 30/10/2018.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+import Foundation
+import Networking
+
+class SuggestionCollectionViewController: UICollectionViewController {
+    
+    private let _image = UIImage(named: "book")
+    private let _headerID = "header"
+    private let _BookCell = "SuggestionCell"
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        collectionView!.register(UINib(nibName: "SuggestionBookCollectionCell", bundle: nil), forCellWithReuseIdentifier: _BookCell)
```

Comment:
you should be consistent in using `!` or `?` to access a nullable property. In this case as your need to have a collection, you can set a guard after `super.viewDidLoad()` to ensure that you have one.
```
guard let collectionView = collectionView else {
        return
    }
```